integration_tests/serverless.yml

```diff
@@ -125,7 +125,7 @@ functions:
     runtime: python3.8
     handler: datadog_lambda.handler.handler
     layers:
-      - arn:aws:lambda:${self:provider.region}:464622532012:layer:Datadog-Python38:${env:PYTHON_LAYER_VERSION}
```

Comment:
Is there a reason we swapped to the staging layers?

Reply:
I could not find a way to dynamically get the latest versions of prod layers without adding prod credentials as secrets